manifests/c/Chromium/ChromeDriver/123.0.6312.58/Chromium.ChromeDriver.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Chromium.ChromeDriver
+PackageVersion: 123.0.6312.58
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: chromedriver.exe

```

```suggestion
- RelativeFilePath: chromedriver-Win64/chromedriver.exe
```
The driver appears to have moved to within the `Win64` folder - does this imply that the `x86` version has been depreciated?